java/README.md

```diff
@@ -0,0 +1,32 @@
+# API Example
+
+Example app exercising some of Climate's [FieldView API](https://dev.fieldview.com).
+
+## Setup
+
+1. Install [Java 8+](https://www.java.com/en/download/)
+2. Install [Gradle](https://gradle.org/install/)
+3. Set the following environment variables
+
+```bash
+export CLIMATE_APP_ID="my-api-id"
+export CLIMATE_APP_SECRET="azbq56fpadhnt8oukoeani2a4w"
+export CLIMATE_API_KEY="my-api-id-216b9875-0158-4142-1ab2-7c3bdbd6a2157"
+export CLIMATE_API_SCOPES=" fields:read asPlanted:read"
```

Comment:
Minor - Do we want to consider using SAMPLE_ prefix instead of CLIMATE_?  
CLIENT_ID and CLIENT_SECRET instead of APP_ID and APP_SECRET?
"my_client_id" instead of "my_api_id"?

Reply:
Yes, make sense.